debian/changelog

```diff
@@ -1,3 +1,15 @@
+daos (2.7.100-6) unstable; urgency=medium
```

Comment:
Hmm. This isn't quite right. I would recommend dropping the changes to `debian/changelog` and the changelog section of `daos.spec`. Otherwise, we're going to get weird conflicts on future merges.

Reply:
Fixed.